lib/services/database_service.dart

```diff
@@ -106,21 +126,31 @@ class DatabaseService {
       _firestoreSubscriptions.conversations =
           _firestore.connectToConversations(userId, _storeController);
     } catch (error, trace) {
-      _storeController.add(AddProblem.from(
-          message: error.toString(),
-          traceString: trace.toString(),
-          type: ProblemType.createConversation));
+      _storeController.add(
+        AddProblem(
+          problem: CreateConversationProblem.by(
```

Comment:
Should this be "ObserveConversationProblem"?

Reply:
Definitely! In the latest commit